### PR TITLE
[ADF-4226]Adding tests for infinite pagination.

### PR DESCRIPTION
**Pull request by cristinaj**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4226

### CODE REVIEW

**Comment by eromano** [comment]
C299201 seems to e failing

File: e2e/core/infinite-scrolling.e2e.ts [hunk 1]
```diff
@@ -73,9 +82,49 @@ describe('Enable infinite scrolling', () => {
...
         contentServicesPage.doubleClickRow(folderModel.name);
         contentServicesPage.enableInfiniteScrolling();
-        contentServicesPage.clickLoadMoreButton();
+        infinitePaginationPage.clickLoadMoreButton();
         for (fileNum; fileNum < nrOfFiles; fileNum++) {
             contentServicesPage.checkContentIsDisplayed(fileNames[fileNum]);
         }
     });
+
+    it('[C299201] Should use default pagination settings for infinite pagination', function () {
```
Comment (marouanbentaleb) [inline e2e/core/infinite-scrolling.e2e.ts]: Can we use '() =>' instead of 'function ()' ?

File: e2e/core/infinite-scrolling.e2e.ts [hunk 2]
```diff
@@ -73,9 +82,49 @@ describe('Enable infinite scrolling', () => {
...
+        contentServicesPage.enableInfiniteScrolling();
+        expect(contentServicesPage.numberOfResultsDisplayed()).toBe(pageSize);
+        infinitePaginationPage.clickLoadMoreButton();
+        expect(contentServicesPage.numberOfResultsDisplayed()).toBe(nrOfFiles);
+
+        infinitePaginationPage.checkLoadMoreButtonIsNotDisplayed();
+        contentServicesPage.checkPaginationIsNotDisplayed();
+    });
+
+    it('[C299202] Should not display load more button when all the files are already displayed', function () {
```
Comment (marouanbentaleb) [inline e2e/core/infinite-scrolling.e2e.ts]: Same as above

File: e2e/core/infinite-scrolling.e2e.ts [hunk 3]
```diff
@@ -73,9 +82,49 @@ describe('Enable infinite scrolling', () => {
...
+        contentServicesPage.checkAcsContainer();
+        contentServicesPage.doubleClickRow(folderModel.name);
+
+        contentServicesPage.enableInfiniteScrolling();
+        expect(contentServicesPage.numberOfResultsDisplayed()).toBe(nrOfFiles);
+
+        infinitePaginationPage.checkLoadMoreButtonIsNotDisplayed();
+    });
+
+    it('[C299203] Should not display load more button when a folder is empty', function () {
```
Comment (marouanbentaleb) [inline e2e/core/infinite-scrolling.e2e.ts]: Same as above

**Pull request merged**